machine/v1beta1/types_vsphereprovider.go

```diff
@@ -70,6 +70,13 @@ type VSphereMachineProviderSpec struct {
 	// When using LinkedClone, if no snapshots exist for the source template, falls back to FullClone.
 	// +optional
 	CloneMode CloneMode `json:"cloneMode,omitempty"`
+	// Disks is a list of non OS disks to be attached to the VM.
+	// +openshift:enable:FeatureGate=VSphereMultiDisk
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate="",maxItems=0
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate=VSphereMultiDisk,maxItems=15
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf | size(self) == size(oldSelf)",message="additional disks cannot be added or removed once set"
+	// +optional
+	Disks []*VSphereDisk `json:"disks,omitempty"`
```

Comment:
Why `[]*VsphereDisk` instead of `[]VSphereDisk`?

Reply:
Just my preference.  I can change to not be pointer if you prefer.

Reply:
I removed pointer.